pkg/cli/scan.go

```diff
@@ -626,7 +626,13 @@ func resolveInputForRemoteTarget(ctx context.Context, input string) (downloadedA
 		})
 
 		if len(nameMatches) == 0 {
-			return nil, nil, fmt.Errorf("no Wolfi package found with name %q in arch %q", input, arch)
+
+			if arch != "aarch64" {
+				return nil, nil, fmt.Errorf("no Wolfi package found with name %q in arch %q", input, arch)
+			}
+
+			logger.Warnf("no Wolfi package found with name %q in arch %q but will continue, it might not have a package built for this arch.", input, arch)
+			continue
```

Comment:
I think this idea makes sense, and logging a warning in this case is a great call.

Could we make it architecture agnostic? If any arch is missing, but we still find at least one, proceed with scanning but log a warning. If no arches found, error out.

Reply:
done

PTAL